packages/contracts-bedrock/test/L1/L2OutputOracle.t.sol

```diff
@@ -511,4 +489,213 @@ contract L2OutputOracleUpgradeable_Test is CommonTest {
+        l2OutputOracle.initialize({
+            _submissionInterval: submissionInterval,
+            _l2BlockTime: l2BlockTime,
+            _startingBlockNumber: startingBlockNumber,
+            _startingTimestamp: block.timestamp + 1,
+            _proposer: proposer,
+            _challenger: challenger,
+            _finalizationPeriodSeconds: finalizationPeriodSeconds
+        });
+    }
+
+    /// @dev Tests that initialize sets the initial values correctly.
+    function testFuzz_initialize_succeeds(
+        uint256 _submissionInterval,
+        uint256 _l2BlockTime,
+        uint256 _startingBlockNumber,
+        uint256 _startingTimestamp,
+        address _proposer,
+        address _challenger,
+        uint256 _finalizationPeriodSeconds
+    )
+        external
+    {
+        vm.assume(_submissionInterval > 0);
+        vm.assume(_l2BlockTime > 0);
+        vm.assume(_startingTimestamp <= block.timestamp);
+
+        // Reset the initialized field in the 0th storage slot
+        // so that initialize can be called again.
+        vm.store(address(l2OutputOracle), bytes32(uint256(0)), bytes32(uint256(0)));
+        l2OutputOracle.initialize({
+            _submissionInterval: _submissionInterval,
+            _l2BlockTime: _l2BlockTime,
+            _startingBlockNumber: _startingBlockNumber,
+            _startingTimestamp: _startingTimestamp,
+            _proposer: _proposer,
+            _challenger: _challenger,
+            _finalizationPeriodSeconds: _finalizationPeriodSeconds
+        });
+
+        assertEq(l2OutputOracle.SUBMISSION_INTERVAL(), _submissionInterval);
+        assertEq(l2OutputOracle.submissionInterval(), _submissionInterval);
+        assertEq(l2OutputOracle.L2_BLOCK_TIME(), _l2BlockTime);
+        assertEq(l2OutputOracle.l2BlockTime(), _l2BlockTime);
+        assertEq(l2OutputOracle.latestBlockNumber(), _startingBlockNumber);
+        assertEq(l2OutputOracle.startingBlockNumber(), _startingBlockNumber);
+        assertEq(l2OutputOracle.startingTimestamp(), _startingTimestamp);
+        assertEq(l2OutputOracle.PROPOSER(), _proposer);
+        assertEq(l2OutputOracle.proposer(), _proposer);
+        assertEq(l2OutputOracle.CHALLENGER(), _challenger);
+        assertEq(l2OutputOracle.challenger(), _challenger);
+        assertEq(l2OutputOracle.finalizationPeriodSeconds(), _finalizationPeriodSeconds);
+        assertEq(l2OutputOracle.FINALIZATION_PERIOD_SECONDS(), _finalizationPeriodSeconds);
+    }
+
+    /// @dev Tests that initialize reverts if the submissionInterval is zero.
+    function testFuzz_initialize_submissionInterval_reverts(
+        uint256 _submissionInterval,
+        uint256 _l2BlockTime,
+        uint256 _startingBlockNumber,
+        uint256 _startingTimestamp,
+        address _proposer,
+        address _challenger,
+        uint256 _finalizationPeriodSeconds
+    )
+        external
+    {
+        vm.assume(!(_submissionInterval > 0));
```

Comment:
You could simplify this expression by flipping to `<` and remove the `!`?

Reply:
this was intentional to show that a line identical to the requirement (originally `_submissionInterval > 0`) is assumed for the contrary (by just adding `!` at the beginning). should I simplify this expression by flipping to `<` and remove the `!`?